### ISSUE TITLE
[BUG] Can't install latest version of npm using npm -> method not allowed

### ISSUE THREAD

**Issue by aiwha-dev** [npm/cli#2608]
### Current Behavior:

Run the following command: npm install -g npm

Since the release of yesterday it gives:
`npm ERR! Error: Method Not Allowed`

Full traceback
```
npm WARN engine npm@7.5.2: wanted: {"node":">=10"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN package.json minipass-collect@1.0.2 No repository field.
npm WARN package.json minipass-pipeline@1.2.4 No repository field.
npm WARN package.json promise-all-reject-late@1.0.1 No repository field.
npm ERR! Error: Method Not Allowed
npm ERR!     at errorResponse (/usr/share/npm/lib/cache/add-named.js:260:10)
npm ERR!     at /usr/share/npm/lib/cache/add-named.js:203:12
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:167:7)
npm ERR!     at Object.oncomplete (fs.js:107:15)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>
```

### Expected Behavior:

To install the latest version

### Steps To Reproduce:

Install npm on a linux distro using yum,
then update it using npm

**Comment by darcyclarke** [comment]
@SamanthaAdrichem sorry to hear you're having issues, unfortunately, we don't support Node `v0.10`. You'll need to update your version of Node to v10 or later to be able to use the latest `npm`

**Comment by ljharb** [comment]
@SamanthaAdrichem if you must use node v0.10, i suggest using https://github.com/nvm-sh/nvm - you can run `nvm install 0.10 --latest-npm` and you'll get the latest version of npm that works on that node version (which isn't v7, but is later than v1)

**Comment by aiwha-dev** [comment]
We don’t need that version, the error persisted with version 14.15.1 aswell we just have a docker image and that installs node, then we use node to upgrade node and npm. Just in a reversed order. It worked fine until the release was made 9 days ago. Then it became method not allowed. Setting it fixed to the version that comes with node 14.15.1 solved it. But it shouldn’t give the error method not allowed..

i will grab an old pipeline and reverse the two and copy the error again if the versions are different there. On monday. It’s weekend here and late now. 

No rush tho, we already have solved it for us. I figured it was a permission issue at your side since it says method not allowed while i can install 6.x

**Comment by ljharb** [comment]
@SamanthaAdrichem do you perhaps have an internal npm registry? or are you talking to the public one
